ui/components/multichain/edit-networks-modal/edit-networks-modal.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
```

Comment:
looks like lint is the only check left to make it green
<img width="1413" alt="Screenshot 2024-07-29 at 6 40 10 PM" src="https://github.com/user-attachments/assets/83da5fac-43eb-4a56-8e61-3fc78e3dd9a2">